applications/MeshingApplication/custom_processes/mmg_process.cpp

```diff
@@ -732,6 +758,21 @@ void MmgProcess<TDim>::ExecuteRemeshing()
     /* After that we reorder nodes, conditions and elements: */
     ReorderAllIds();
     
+    /* Unmoving the original mesh to be able to interpolate */
+    if (mFramework == Lagrangian) 
```

Comment:
i think that the Enum name is misleading. If it is UpdatedLagrangian you may well interpolate on the latest configuration. If you are Total lagrangian than you may need to do what you are doing here

Reply:
I am using the original configuration for search the nodes, the interpolated values are the ones from the last configuration